src/main/scala/FundingMode/Calculation.scala

```diff
@@ -0,0 +1,30 @@
+package FundingMode
+
+import FundingMode.CaseClassDeclaration._
+
+object Calculation {
+  val prohibited  = "prohibited"
+  val collections = "collections"
+  val receipts    = "receipts"
```

Comment:
These 3 value stand for FundingType and you can have only 3 types, so we can treat that as enum. In Scala you can use sealed trait for that purpose:
`sealed trait FundingyType 
 case object Prohibited extends FundingyType
case object Collections  extends FundingyType
case object Receipts extends FundingyType
`